### ISSUE TITLE
PieLauncher as accessibility app

### ISSUE THREAD

**Issue by pacjo** [markusfisch/PieLauncher#33]
I was wondering is it is possible for PieLauncher to be used as an accessibility app (for example for quick shortcuts).

It would be possible to use android built-in accessibility gestures (shown below) to launch pielauncher and than using it quickly open one of preselected apps.

[KISS Launcher](https://github.com/Neamar/KISS) already implements accessibility features, so I was wondering if it would be possible to add them here too.

![Screenshot_20220315-141741_Settings](https://user-images.githubusercontent.com/56438628/158386202-9f1015e9-3472-46d4-aef3-948c8746294f.png)

**Comment by markusfisch** [comment]
Yes, probably 🤔 I will have to learn more about these gestures and how they should interact with a launcher.
